capa/v29.1.0/README.md

```diff
@@ -0,0 +1,48 @@
+# :zap: Giant Swarm Release v29.1.0 for CAPA :zap:
+
+## Changes compared to v29.0.0
+
+### Components
+
+- Flatcar from v3815.2.5 to [v3975.2.0](https://www.flatcar.org/releases#release-3975.2.0)
+- Kubernetes from v1.29.7 to [v1.29.8](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.29.md#changelog-since-v1297)
+- os-tooling from v1.15.0 to v1.18.1
```

Comment:
Should we exclude this internal chart detail? Customers can't easily find out what this is.

Reply:
Absolutely, removed the os-tooling line with the latest commit :+1: